armi/physics/fuelCycle/tests/test_fuelHandlers.py

```diff
@@ -60,7 +60,7 @@ def setUp(self):
         but none of these have any number densities.
         """
         self.o, self.r = test_reactors.loadTestReactor(
-            self.directoryChanger.destination
+            self.directoryChanger.destination, customSettings={"nCycles": 3}
```

Comment:
This one was an anomaly. For whatever reason, the rest of this test was set up under the assumption that there would be 3 cycles. But the input that is loaded says 6 cycles.

That input is used for a bunch of other tests, so I don't want to change that.

And the rest of this test is already set up to expect 3 cycles. So I don't want to change the rest of the tests.

I can't remember exactly how it was working before, but I do remember that there were a few places where this type of thing was getting tangled up, that I had to sort out. For whatever reason, it just landed this way that this test needed to be fixed.

But all the other places that previously had similar `customSettings` being passed in have since been removed, so this is the only one left.